Add /builds page

pearswj opened the pull request:
Going back to the [slack discussion](https://speckle-works.slack.com/archives/C4U5UM3EY/p1526998306000669) about making build artifacts more accessible...

First pass at porting the script I wrote for https://pearswj.co.uk/builds/compat from travis-ci to AppVeyor. Displays latest builds for SpeckleRhino by pulling data from the AppVeyor API.

A few thoughts/caveats...

* It's a little slow because the API requires a call per `build` to get the `jobId` for the artifact download links.
  * Could be sped up by just linking to the [artifacts tab on the build page](https://ci.appveyor.com/project/SpeckleWorks/specklerhino/build/367/artifacts). (Might be necessary if/when this page shows builds from other projects too.)
  * AppVeyor now has a [3 month artifact retention policy](https://www.appveyor.com/blog/2018/05/24/artifacts-retention-policy/), so we might consider pushing build artifacts elsewhere (again, saving many API calls to get the direct download link for artifacts).
* I've included the latest 10 builds from the `master` branch and pull requests only.
* It doesn't have to be styled in this monospaced, tabular layout.

Open to suggestions!

![screen shot 2018-06-03 at 10 26 18](https://user-images.githubusercontent.com/121068/40885195-7c55c5d0-6719-11e8-9671-2787223ac183.png)

/cc @didimitrie @fraguada @Eckart-S

pearswj committed:
Add /builds page

Displays latest builds for SpeckleRhino by pulling data from the
AppVeyor API.

didimitrie commented:
I like it! Monospace FTW! 

Questions/suggestions:
- does it make sense to add `SpeckleCore` master builds under a separate heading, eg .NET SDK?
- add SpeckleDynamo heading with 'coming soon...' 😅under it, and once we've got that CI'ed up, plop the Dynamo builds in there? 

And most importantly: why not link [it in the top menu too](https://github.com/speckleworks/speckleworks-website/blob/master/config.toml#L9)? 

Otherwise, re bullet point number one - we'll sort those out in time. It would make sense to push builds to a different place - but, how, where and who'll pay are some tough questions for that at the moment 😬 (also adds pressure to put up a build at least every 3 months!)

pearswj committed:
Tweak builds page

* Adds nav bar entry
* Adds builds for SpeckleCore and placeholder for SpeckleDynamo
* Uses generic artifact link to save loading time

pearswj commented:
![screen shot 2018-06-04 at 08 59 12](https://user-images.githubusercontent.com/121068/40906660-620ae686-67d9-11e8-892b-5bdc59d67c8f.png)

pearswj committed:
Use file archive glyph instead of download

pearswj commented:
Quick glyph change... 😬 

![screen shot 2018-06-04 at 09 39 23](https://user-images.githubusercontent.com/121068/40907282-39e07296-67db-11e8-86c9-5860cb6543c4.png)

pearswj committed:
Add /builds page to menu via frontmatter instead

Makes `active` styling work. See
https://discourse.gohugo.io/t/ismenucurrent-failing-for-me-outside-of-the-home-page/11461/2

pearswj commented:
Apparently there's a bug in Hugo that prevents `$currentPage.IsMenuCurrent` from working for custom pages. Fixed by [adding the page to the menu via the frontmatter](https://discourse.gohugo.io/t/ismenucurrent-failing-for-me-outside-of-the-home-page/11461/2), instead of `config.toml`.

![screen shot 2018-06-04 at 11 55 10](https://user-images.githubusercontent.com/121068/40913918-2e1869a6-67ee-11e8-959a-4a22a5112c69.png)

That's it. I'm happy with this if you are @didimitrie!

didimitrie commented:
Merge & `deploy.sh` away!
